sql/hive/src/test/scala/org/apache/spark/sql/hive/CachedTableSuite.scala

```diff
@@ -204,6 +204,61 @@ class CachedTableSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
     }
   }
 
+  test("Refresh Qualified Tables") {
+    withTempDatabase { db =>
+      withTempView("cachedTable") {
+        try {
+          sql(s"CREATE TABLE $db.cachedTable STORED AS PARQUET AS SELECT 1")
+          sql(s"CACHE TABLE $db.cachedTable")
+          assertCached(sql(
```

Comment:
Nit: indentation. Move `sql(` to the beginning of the next line.

Reply:
sql was just move to the next line.